### PR TITLE
Title was missing

### PR DESCRIPTION
**Pull request by mwijns**

### CODE REVIEW

File: src/main/markdown/doc/latest/DevGuideCompileReport.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Compile Reports
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/DevGuideCompileReport.md]: `Compilation reports` or `compiler reports` don't sound better ?

Comment (mwijns) [inline src/main/markdown/doc/latest/DevGuideCompileReport.md]: I agree with you; I went for consistency with the rest of the site, although Compilation reports sounds better.

Comment (jDramaix) [inline src/main/markdown/doc/latest/DevGuideCompileReport.md]: Could you make the change ?

**Comment by jDramaix** [comment]
friendly ping

**Comment by mwijns** [comment]
Sorry, didn't get to it and know I'm enjoying a 6-months leave, so no, not
any time soon.

2015-03-18 19:25 GMT+01:00 Julien Dramaix <notifications@github.com>:

**Comment by jDramaix** [comment]
Ok let merge this and I will do the change myself.

**Pull request merged**